build.gradle.kts

```
@@ -66,7 +66,7 @@ base {
 
 loom {
     refmapName = "galacticraft-rewoven.refmap.json"
-    accessWidener = File("src/main/resources/galacticraft_rewoven.accesswidener")
+    accessWidener = File(project.buildDir.toString() + "/../src/main/resources/galacticraft_rewoven.accesswidener")
```

Can you test if this works for you? (I'd like to avoid the string concatenation)
```suggestion
    accessWidener = rootProject.file("src/main/resources/galacticraft_rewoven.accesswidener")
```

---

It should and it does